STUDIO/org.openl.rules.repository/src/org/openl/rules/repository/api/MergeConflictException.java

```diff
@@ -1,20 +1,25 @@
 package org.openl.rules.repository.api;
 
+import org.openl.rules.xls.merge.XlsDiffResult;
+
 import java.io.IOException;
 import java.util.Collection;
+import java.util.Collections;
 import java.util.Map;
 
 public class MergeConflictException extends IOException {

```

Comment:
Move this class to the repository.git module

Reply:
Done